File: Python /CashDispenser.py

class CashDispenser: 
    INITIAL_COUNT = 500; 
    def __init__(self):
        self.count = self.INITIAL_COUNT; 

    def dispense_cash(self, amount):
        bills_required = amount // 20; 
        self.count -= bills_required; 

    def is_sufficient_cash_available(self, amount):
        bills_required = amount // 20; 
        if self.count >= bills_required:
            return True; 
        else:
            return False;

File: Golang/CashDispenser.go
package main

type CashDispenser struct {
	count int
}

const INITIAL_COUNT = 500

func (c *CashDispenser) DispenseCash(amount int) {
	billsRequired := amount / 20
	c.count -= billsRequired
}

func (c *CashDispenser) IsSufficientCashAvailable(amount int) bool {
	billsRequired := amount / 20
	if c.count >= billsRequired {
		return true
	} else {
		return false
	}
}

func NewCashDispenser() *CashDispenser {
	return &CashDispenser{count: INITIAL_COUNT}
}
